### PR TITLE
Fail on exceeding threshold

### PR DESCRIPTION
**Pull request by adithyaov**

### CODE REVIEW

File: benchmark/bench-report/bin/bench-runner.sh [hunk 1]
```diff
@@ -18,6 +18,7 @@ print_help () {
   echo "       [--benchmarks <"bench1 bench2 ..." | help>]"
   echo "       [--prefix <benchmark name prefix to match>"
   echo "       [--fields <"field1 field2 ..." | help>]"
+  echo "       [--failure-field]"
```
Comment (harendra-kumar) [inline benchmark/bench-report/bin/bench-runner.sh:21]: We can call it `--fail-on`.

File: benchmark/bench-report/bin/bench-runner.sh [hunk 2]
```diff
@@ -40,6 +41,8 @@ print_help () {
   echo "--benchmarks: benchmarks to run, use 'help' for list of benchmarks"
   echo "--compare: compare the specified benchmarks with each other"
   echo "--fields: measurement fields to report, use 'help' for a list"
+  echo "--threshold: threshold percentage to decide regression"
```
Comment (harendra-kumar) [inline benchmark/bench-report/bin/bench-runner.sh:44]: We already have `--diff-cutoff-percent` that serves a similar purpose.

File: benchmark/bench-report/bin/bench-runner.sh (hunk 1 above)
Comment (adithyaov) [reply benchmark/bench-report/bin/bench-runner.sh:21]: We probably don't need this. It should fail on all the fields. We can filter fields accordingly though.

**Comment by adithyaov** [comment]
See #1431

**Pull request closed without merging**